codespell_lib/data/dictionary.txt

```diff
@@ -14016,6 +14021,9 @@ reesrved->reserved
 reesult->result
 reevalute->reevaluate
 reevaulating->reevaluating
+refartor->refactor, refractor,
+refartored->refactored, refactored,
```

Comment:
Did you mean refracted? It's currently a duplicate word as per Travis.

Reply:
Ooops :blush:  
Thanks

Reply:
8a48688